tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/GraphQLTests.cs

```diff
@@ -144,6 +144,178 @@ protected GraphQLTests(string sampleAppName, AspNetCoreTestFixture fixture, ITes
 
         protected AspNetCoreTestFixture Fixture { get; }
 
+        public static void SubmitRequest(ITestOutputHelper output, int aspNetCorePort, RequestInfo requestInfo, bool printResponseText = true)
```

Comment:
If this is going to be shared by GraphQL & HotChocolate, I think would be good to put it in a separate class, something like `GraphQL` common.

Reply:
I can create an other class but, where do I create this class? Inside the IntegrationTests solution with all other Tests class?